capitolwords_ng/parsers/text_utils.py

```diff
@@ -86,28 +145,64 @@ def get_named_entities(doc, exclude_types=NUMERIC_NE_TYPES, drop_determiners=Tru
         drop_determiners (bool)
 
     Returns:
-        list of :class:`spacy.Span()`
+        list of tuple (:class:`spacy.Span()`, str)
```

Comment:
I'm still not completely familiar with rst object stuff, what does including the parens at the end do?